### PR TITLE
Doc update for 3.1 release

### PR DESCRIPTION
**Pull request by shivamerla**
Signed-off-by: Shiva Krishna, Merla <shivakrishna.merla@hpe.com>

### CODE REVIEW

File: release-notes/v3.1.0.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# HPE Volume Driver for Kubernetes FlexVolume Plugin 3.1.0 Release Notes
+
+## HPE Volume Driver for Kubernetes FlexVolume Plugin 3.1.0
+
+| **Version:** |3.1.0|
+|--------------|-----|
+| **Revision:** | Friday Nov 1, 2019 22:05:17|
+
+The release notes describe the major changes, fixes, and known issues for this release of the new for HPE Volume Driver for Kubernetes FlexVolume Plugin 3.1.0. They do not include all individual fixes and internal changes.
```
Comment (rgcostea) [inline release-notes/v3.1.0.md]: rephrase:
for this release of the new for HPE...

File: release-notes/v3.1.0.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+| **Critical:**  |HPE Nimble Storage continues to qualify configurations between releases. The Validated Configuration Matrix provides information about validated configurations and is updated frequently. It is a good practice to check your system configuration against this online tool. The Validated Configuration Matrix tool is available on HPE InfoSight:  [https://infosight.hpe.com/resources/nimble/validated-configuration-matrix](https://infosight.hpe.com/resources/nimble/validated-configuration-matrix)|
+| **Important:**  |HPE Nimble Storage Volume Driver for Kubernetes 3.1.0 supports multiple versions of NimbleOS starting with NimbleOS 5.1.3.x. Details about this tool are available on the HPE Nimble Storage Github at [https://github.com/hpe-storage/flexvolume-driver/blob/master/README.md](https://github.com/hpe-storage/flexvolume-driver/blob/master/README.md).|
+
+## Documentation
+
+The documentation for HPE Nimble Storage and HPE CloudVolumes Volume Plugin for Docker is available on the HPE Nimble Storage Github at [https://github.com/hpe-storage/flexvolume-driver/blob/master/README.md](https://github.com/hpe-storage/flexvolume-driver/blob/master/README.md).
```
Comment (rgcostea) [inline release-notes/v3.1.0.md]: Cloud Volumes

**Review by rgcostea — APPROVED** [review]

File: release-notes/v3.1.0.md (hunk 2 above)
Comment (shivamerla) [reply release-notes/v3.1.0.md]: done..thanks.

File: release-notes/v3.1.0.md (hunk 1 above)
Comment (shivamerla) [reply release-notes/v3.1.0.md]: done..

**Pull request merged**